integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/framework/runner/Utils.kt

```diff
@@ -24,11 +28,52 @@ internal fun GradleBuild.buildGradleByShell(
     return BuildResult(exitCode, logFile)
 }
 
+internal fun Path.enableCacheRedirector() {
+    // Path relative to the current gradle module project dir
+    val redirectorScript =
+        Paths.get("../build-settings-logic/src/main/kotlin/atomicfu-cache-redirector.settings.gradle.kts")
+            .toAbsolutePath()
+            .normalize()
+            .toFile()
```

Comment:
For Gradle up-to-date checks this file should be registered as a test task input.

Reply:
I'm unsure if it is necessary because the cache-redirector is only responsible for downloading dependencies.
When the task is up-to-date, we don't need to download dependencies, but if we change smth in cache-redirector, we also don't need to rerun the task. Or am I wrong?

Reply:
also, [here](https://github.com/Kotlin/kotlinx-atomicfu/blob/dc1c5f5162dd976a9bded61d0d4b967c05d719ae/integration-testing/build.gradle.kts#L91) we have 
```kotlin
outputs.upToDateWhen { false }
```
which as I understand always forces to run `functionalTest`

Reply:
moved path to system property